src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -696,16 +699,10 @@ internal static bool GenerateSerializer(Type[]? types, XmlMapping[] mappings, St
         private static XmlSerializer[] GetSerializersFromCache(XmlMapping[] mappings, Type type)
         {
             XmlSerializer?[] serializers = new XmlSerializer?[mappings.Length];
-
             Dictionary<XmlSerializerMappingKey, XmlSerializer>? typedMappingTable = null;
-            lock (s_xmlSerializerTable)
-            {
-                if (!s_xmlSerializerTable.TryGetValue(type, out typedMappingTable))
-                {
-                    typedMappingTable = new Dictionary<XmlSerializerMappingKey, XmlSerializer>();
-                    s_xmlSerializerTable[type] = typedMappingTable;
-                }
-            }
+            AssemblyLoadContext? alc = AssemblyLoadContext.GetLoadContext(type.Assembly);
```

Comment:
This doesn't seem to be used anywhere.

Reply:
Ahh, a remnant from a previous iteration where we spread logic out instead of encapsulating these repetitive patterns in our new ContextAwareTable. I will remove this line.